### PR TITLE
Add CIS 3.1.2, 3.2.1, 3.2.2

### PR DESCRIPTION
**Pull request by satya-rajesh**
verified both locally and on docker

### CODE REVIEW

File: roles/cis/tasks/3/3.1.1.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@
             touch {{ security_hardening_conf }}
         fi
       delegate_to: localhost
```
Comment (ppadmavilasom) [inline roles/cis/tasks/3/3.1.1.yml]: can we remove this?

File: roles/cis/tasks/3/3.1.1.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@
         reload: no
         ignoreerrors: yes
       delegate_to: localhost
-    - name: "set net.ipv6.conf.all.forwarding"
+    - name: "3.1.1 set net.ipv6.conf.all.forwarding to 0"
       sysctl:
         name: net.ipv6.conf.all.forwarding
         value: 0
```
Comment (ppadmavilasom) [inline roles/cis/tasks/3/3.1.1.yml]: quote all these values please. value: '0'

**Review by ppadmavilasom — COMMENTED** [review]
overall, i dont think these need to be delegated to localhost. they can transparently apply chroot or otherwise.

File: roles/cis/tasks/3/3.1.1.yml (hunk 1 above)
Comment (satya-rajesh) [reply roles/cis/tasks/3/3.1.1.yml]: Pre-existing /etc/sysctl.d/50-security-hardening.conf is outside of chroot, i.e., this file is in photon vm. If it is not delegated to localhost, it complains that file is not accessible/not present.
If written to /etc/sysctl.conf did not see any issues.

File: roles/cis/tasks/3/3.1.1.yml (hunk 2 above)
Comment (satya-rajesh) [reply roles/cis/tasks/3/3.1.1.yml]: I will check this and update you.

**Comment by satya-rajesh** [comment]
> overall, i dont think these need to be delegated to localhost. they can transparently apply chroot or otherwise.

Same observation as above, not able to access pre-existing file /etc/sysctl.d/50-security-hardening.conf with chroot as it is outside chroot. Please let me know your suggestions.

**Comment by ppadmavilasom** [comment]
@satya-rajesh the changes are supposed to happen in the chroot. look in /root/testroot/etc/sysctl.d 
my point is that there is no need to special case

File: roles/cis/tasks/3/3.1.1.yml (hunk 1 above)
Comment (satya-rajesh) [reply roles/cis/tasks/3/3.1.1.yml]: Fixed!

File: roles/cis/tasks/3/3.1.1.yml (hunk 2 above)
Comment (satya-rajesh) [reply roles/cis/tasks/3/3.1.1.yml]: Fixed!

**Comment by satya-rajesh** [comment]
> @satya-rajesh the changes are supposed to happen in the chroot. look in /root/testroot/etc/sysctl.d
> my point is that there is no need to special case

Fixed!

**Review by ppadmavilasom — APPROVED** [review]
LGTM

**Pull request merged**